CMakeCommon.cmake

```diff
@@ -246,3 +251,42 @@ function (GenerateAddOnProject target acVersion devKitDir addOnName addOnSources
     SetCompilerOptions (${target} ${acVersion})
 
 endfunction ()
+
+function (check_valid_language_code configFile languageCode)
+    file (READ "${configFile}" configsContent)
+    string (JSON configuredLanguagesList GET "${configsContent}" "languages")
+    string (JSON configuredLanguagesListLen LENGTH "${configsContent}" "languages")
+    set (i 0)
+    while (i LESS configuredLanguagesListLen)
+        string (JSON language GET "${configuredLanguagesList}" "${i}")
+        if ("${language}" STREQUAL "${languageCode}")
```

Comment:
I'd prefer to see just the variable in such cases where the syntax is `<variable|string>`. Also on line 277.